packages/account/src/Configs/address-details-config.js

```diff
@@ -47,11 +47,12 @@ const address_details_config = ({ account_settings, is_svg }) => {
             default_value: account_settings.address_city ?? '',
             rules: [
                 ['req', localize('City is required')],
+                ['length', localize('This should not exceed {{max}} characters.', { max: 99 }), { max: 99 }],
                 [
                     'regular',
-                    localize('Letters, numbers, spaces, periods, hyphens, apostrophes only'),
+                    localize('Only letters, numbers, spaces, periods, hyphens, and apostrophes allowed'),
```

Comment:
This text doesn't look right, has it been approved by copy team?

Reply:
Text was updated according to content team suggestion